components/org.wso2.carbon.identity.api.server.branding.preference.management/org.wso2.carbon.identity.api.server.branding.preference.management.common/src/main/java/org/wso2/carbon/identity/api/server/branding/preference/management/common/factory/BrandingPreferenceMgtOSGiServiceFactory.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
```

Comment:
Since the class was originally added in 2021, let's change the year back to 2021.

Reply:
Addressed with https://github.com/wso2/identity-api-server/pull/340/commits/1f9c2167a46e2ac87fff186a9cbaab5f1bceb39b